C++/C.sublime-syntax

```diff
@@ -255,10 +157,27 @@ contexts:
     - match: ^\s*#\s*if(n?def)?\b.*$
       comment: eat nested preprocessor if(def)s
       push:
-        - match: ^\s*#\s*endif\b.*$
+        - match: ^\s*(#\s*(endif)\b)
+          captures:
+            1: meta.preprocessor.c
+            2: keyword.control.import.if.c
           pop: true
         - include: disabled
         - include: pragma-mark
+  operators:
+    - match: <\=|>\=|\=\=|<|>|\!\=
+      scope: keyword.operator.comparison.c
+    - match: \+\=|-\=|\*\=|/\=|//\=|%\=|&\=|\|\=|\^\=|>>\=|<<\=|\*\*\=
```

Comment:
@TheClams I don't think the operator highlighting is quite correct. I have a few suggestions:

1. You are highlighting `**`, `**=`, `//`, and `//=` which are neither C nor C++ operators (`//` starts a comment)
2. Since the comparison operators are first, I think they will prevent some of the arithmetic operators from being tagged properly. For example, I think the `<<` operator will be tagged as two `<` operators. I haven't tested this, though, so I don't know for sure. It would be a good idea to add some of these operators to the test file to look into this
3. Could you highlight the `!`, `&&`, and `||` operators as well? Thanks!